tests/python_tests/test_whisper_pipeline.py

```diff
@@ -166,13 +179,44 @@ def get_samples_from_dataset(
     return [x["audio"]["array"] for x in ds]
 
 
-@pytest.mark.parametrize("model_descr", get_whisper_models_list())
-@pytest.mark.parametrize("dataset_id", ["hf-internal-testing/librispeech_asr_dummy"])
-@pytest.mark.precommit
-def test_whisper_on_hf_dataset(model_descr, dataset_id):
-    model_id, path, opt_pipe, genai_pipe = read_whisper_model(model_descr)
+def run_pipeline_with_ref(
+    model_id: str,
+    tmp_path: str,
+    sample: np.ndarray | list[np.ndarray],
+    generation_config: GenerationConfig | None = None,
+):
+    _, _, hf_pipe, genai_pipe = read_whisper_model((model_id, tmp_path))
+
+    if type(sample) is np.ndarray and len(sample.shape) == 1:
+        sample = np.expand_dims(sample, 0)
+
+    for _sample in sample:
+        genai_result = run_genai(genai_pipe, _sample, generation_config)
+        hf_result = run_huggingface(hf_pipe, _sample, generation_config)
+
+        compare_results(hf_result, genai_result)
+
 
-    compare_genai_and_opt_pipelines(opt_pipe, genai_pipe, dataset_id)
+def compare_results(hf_result, genai_result):
+    assert genai_result.texts[0] == hf_result["text"]
+
+    # transformers 4.47 updated return_timestamps implementation
+    # enable once genai implementation aligned with trasformets. Ticket 160205.
+    return
```

Comment:
maybe we need to check transformers version here? and on CI we can use one of the platforms to install older transformers to validate old behavior

Reply:
You mean as a temporary solution until implementations are aligned?

Reply:
yes, but depends on what time is required to align implementations.

if it's easy / fast enough, we can keep current code with skip of validation.

Reply:
I updated Win workflow to install 4.46.3 transformers. Also added check of transformers version in order to validate timestamps